index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+        integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&family=Roboto:wght@500&display=swap"
+        rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>-Portfolio-</title>
+</head>
+
+<body>
+    <header>
+        <div id="statusbar">
+            <div>9:42</div>
+            <div id="status-icons">
+                <i class="fa-solid fa-signal"></i>
+                <i class="fa-solid fa-wifi"></i>
+                <i class="fa-solid fa-battery-full"></i>
+            </div>
+        </div>
+        <div id="toolbar">
+            <div id="logo"><a href="#">My logo</a></div>
+            <nav>
+                <div id="menu"><i class="fa-solid fa-bars"></i></div>
+            </nav>
+        </div>
+    </header>
+
+    <section>
+        <div class="column">
+            <span>
+                <div id="img-container"></div>
+            </span>
+            <h1>I'm Julio. Glad to see you!</h1>
+            <p>I’m a software developer! I can help you build a product , feature or website Look through some of my
+                work and experience! If you like what you see and have a project you need coded, don’t hesitate to
+                contact me.</p>
+            <div id="social-media-icons">
+                <p>LET'S CONNECT</p>
+                <ul>
+                    <a href="#">
+                        <i class="fa-brands fa-twitter"></i>
+                    </a>
+                    <a href="#"><i class="fa-brands fa-linkedin-in"></i></a>
+                    <a href="#"><i class="fa-sharp fa-solid fa-m"></i></a>
+                    <a href="#"><i class="fa-brands fa-github"></i></a>
+                    <a href="#"><i class="fa-regular fa-hand-peace"></i></a>
+                </ul>
```

Comment:
- [ ] Good job @Roathy 👍, however,  as mentioned buy the previous reviewer,  The list of icons should use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).
Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.

eg:  
````
<ul>
         <li><a href="#">...</a></li>
         <li><a href="#">...</a></li>
         <li><a href="#">...</a></li>
 </ul>
````